.env.development

```diff
@@ -1,2 +1,3 @@
 REACT_APP_GRAPHQL_URL=http://localhost:8002/graphql
```

Comment:
I think there should be one URL labeled: REACT_APP_GITHUB_WIDGET_API which should be http://localhost:8002/ in .env.development for dev purposes and the correct thing for prod in .env.production. All the other urls can be figured out by adding the correct end point to the end of the this url .

Reply:
@Gwillison415 I would love for you to do the refactor I'm asking for, in this PR. this will mean less work when integrating this into dashboard-spa. Of course it means some refactoring right now upfront. Alongside this please submit a PR which would take care of this env variable stuff in dashboard-spa. thanks

Reply:
sure thing i'm on it!

Reply:
given our conversation  - `${process.env.REACT_APP_GITHUB_WIDGET_API}` will be in a new PR. thanks for that